### PR TITLE
Reduce storage and fix internally draining gaps.

### PR DESCRIPTION
**Pull request by dtarb**
This addresses the issue #53

### CODE REVIEW

File: rwd_nhd/NHD_Rapid_Watershed_Delineation.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import sys
 import os
 import time
+import subprocess
```
Comment (dtarb) [inline rwd_nhd/NHD_Rapid_Watershed_Delineation.py]: Could exclude this if not using subprocess on linux

File: rwd_nhd/NHD_Rapid_Watershed_Delineation.py [hunk 2]
```diff
@@ -82,27 +104,52 @@ def Point_Watershed_Function(
...
+        infile= os.path.join(grid_dir,grid_name+"p.tif")
+        outfile="localp.tif"
+        ulx= str(albers_x - 11000.0)
+        lrx=str(albers_x + 11000.0)
+        uly=str(albers_y + 11000.0)
+        lry=str(albers_y - 11000.0)
+        cmd="gdal_translate -projwin " + ulx + " " +  uly + " " + lrx + " " +  lry + ' "' + infile + '" ' + outfile
+        # subprocess.check_call(cmd)
+        os.system(cmd)  
+
```
Comment (dtarb) [inline rwd_nhd/NHD_Rapid_Watershed_Delineation.py]: In PC testing I used the subprocess call.  the os.system call is untested.

**Comment by kdeloach** [comment]
Consider splitting this up into 2 pull requests. One to reduce storage, and one to fix internally draining gaps.

Updating the Python code is a relatively quick process, but updating the underlying data is very time consuming. We'd prefer to wait a little longer before applying the data updates, just in case we discover more updates that need to made-- so we can apply them all at once.

**Comment by dtarb** [comment]
The fix to internally draining gaps involves data updates as it requires the new regions data that I have provided.  The flow directions data to fix the gaps are not in the currently deployed data.

I could separate out python code to just the watershed attribute changes associated with reduced storage without the files being deleted, but this seems hardly worthwhile.

**Comment by kdeloach** [comment]
We were considering not deleting any data (only updating), in case we later decide to use those attributes. From that point of view, it isn't likely that we will need these changes to reduce storage. 

In any case, we can still merge the code in but make a note that we did not necessarily run these scripts on production.

**Comment by kdeloach** [comment]
Using the latest code and data I was able to generate watersheds for areas outlined in the screenshot from #53. 

For future reference, this area is "Rick Husband Amarillo International Airport" which is located East of Amarillo, TX.

Here is a watershed I generated at (roughly) `LatLng(35.21169, -101.69701)`:

screenshot 2017-01-30 at 18 08 38

**Comment by kdeloach** [comment]
This worked without any code changes so I will merge this in tomorrow morning then begin the process to cycle these changes onto the staging environment.

**Pull request merged**